app/controllers/admin/content/system-images/subtopics.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+const { Controller, computed } = Ember;
+export default Controller.extend({
+  subTopics: computed('model.currentTopicID', function() {
```

Comment:
You don't need to return anything here. This whole computed property is not needed. Directly use the model in the template